### ISSUE TITLE
Proposal: Add a group of effects that happen on mouse click events

### ISSUE THREAD

**Issue by 0x000C0A71** [VirtCode/hypr-dynamic-cursors#34]
For example:
- cursor squishes on left-click
- cursor rotates when you scroll

**Comment by 0x000C0A71** [comment]
Is there an ETA for this feature?

**Comment by 0x000C0A71** [comment]
@VirtCode ping

**Comment by bloxx12** [comment]
@VirtCode please, I need this.

**Comment by bloxx12** [comment]
@VirtCode  when will this be added????????????????

**Comment by 0x000C0A71** [comment]
> @VirtCode please, I need this.

That's exactly what I mean

**Comment by bloxx12** [comment]
> > @VirtCode please, I need this.
> 
> That's exactly what I mean

See how much this is needed?
please fix soon, ty.

**Comment by jaredmontoya** [comment]
@VirtCode I think I know why people want this, they want to have osu!lazer UI everywhere

**Comment by 0x000C0A71** [comment]
> @VirtCode I think I know why people want this, they want to rice hyprland to look like osu!lazer UI

Yes ABSOLUTELY i LOVE osu!lazer i LOVE osu!lazer i LOVE osu!lazer i LOVE osu!lazer i LOVE osu!lazer i LOVE osu!lazer i LOVE osu!lazer i LOVE osu!lazer i LOVE osu!lazer

@VirtCode @VirtCode @VirtCode plsplspls

**Comment by VirtCode** [comment]
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

**Comment by bloxx12** [comment]
Stale issue, is this gonna be implemented?
When is the ETA?

**Comment by 0x000C0A71** [comment]
@bloxx12 I get the feeling that @VirtCode is ghosting us to avoid having to implement this highly anticipated feature...

**Comment by littleblack111** [comment]
Bruh just leave the author alone, when they have the time they will implment this, or file a MR ur self

**Comment by 0x000C0A71** [comment]
> Bruh just leave the owner alone, when they have the time they will implment this, or file a MR ur self

nono the owner MUST add this feature ASAP

**Comment by 0x000C0A71** [comment]
@VirtCode

**Comment by 0x000C0A71** [comment]
@VirtCode this is IMPORTANT, remove the `stale` tag **NOW**!

**Comment by daUnknownCoder** [comment]
> That is an awesome cursor theme. Pls share a link to where I can get that! Looks like it would absolutely FLY with click animations! Then we just need to hope @VirtCode doesn't promote this issue to `stalester` or even `stalestest`...

https://github.com/ful1e5/Google_Cursor

currently, @VirtCode be like: 🎶 Oops, I did it again 🎶 (by Britney Spear)

**Comment by 0x000C0A71** [comment]
Nono, you should 100% promote it. This is an *important* feature and we are *blatantly* getting ignored by @VirtCode . We *must not* stay quiet in the face of oppression!

**Comment by Pinuaple** [comment]
@VirtCode COME ON

**Comment by jaredmontoya** [comment]
At this point someone can probably just vibe code it if they want it enough.
Leave GPT 6 Astra or Fable 5.1 overnight on it.

**Comment by Pinuaple** [comment]
someone kind enough with one of those subscriptions pls

omfg the tag descriptions 😭😭😭

**Comment by 0x000C0A71** [comment]
Who here likes Pinuaple on pizza? Pls those in favor react with a thumbs up (👍) and those against with a thumbs down (👎)
